features/upgrade/storage/upgrade.feature

```diff
@@ -404,6 +410,7 @@ Feature: Storage upgrade tests
   @upgrade
   @network-ovnkubernetes @network-openshiftsdn
   @hypershift-hosted
+  @inactive
   Scenario: OCP-44578:Storage AWS-EBS-CSI Snapshot operator should work well before and after upgrade of a cluster - prepare
```

Comment:
The upgrade prepare should remove in pair with the upgrade check.

Reply:
Aha, just realized it when send the last comment, sorry for my overlooking, and thanks for the careful review!